android/app/src/main/java/com/jolocomwallet/MainApplication.java

```diff
@@ -37,13 +39,15 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
-            new SplashScreenReactPackage(),
+            new RNFSPackage(),
+            new RNLanguagesPackage(),
```

Comment:
Why do we need the RNFSPackage() ?

Reply:
Yes, I've forgotten to remove it, thanks.